ghcide.cabal

```diff
@@ -201,10 +202,6 @@ library
         Development.IDE.GHC.HieAst
         Development.IDE.GHC.HieBin
     ghc-options: -Wall -Wno-name-shadowing -Wincomplete-uni-patterns
-    -- This is needed to prevent a GHC crash when building
-    -- Development.IDE.Session with stack on 8.10.1 on Windows
-    if (impl(ghc > 8.9) && os(windows))
-      ghc-options: -fexternal-interpreter
```

Comment:
Why has this been removed?

Reply:
Sorry for nor mentioning it, my local windows build was failing due to a segfault in the external interpreter (the thing that line try to fix in ci!) 
The ci job has failied for other reason, let's see if it continue failing due to segfault without `-fexternal-interpreter` after fixin the actual error.